profiles/keycloak_integration.py

```diff
@@ -91,7 +94,12 @@ def send_profile_changes_to_keycloak(instance):
     if email_changed:
         updated_data["emailVerified"] = False
 
-    _keycloak_admin_client.update_user(user_id, updated_data)
+    try:
+        _keycloak_admin_client.update_user(user_id, updated_data)
+    except requests.HTTPError as err:
```

Comment:
I'm not a big fan of the fact that the "API" for the Keycloak admin client is just requests exceptions.

Could it handle the requests exceptions internally and maybe raise the DataConflictError itself? What do you think?

Reply:
I have been thinking the same thing. The client could definitely communicate errors in some clearer way. Let's do it.

Should I make such change within this PR or create another one for that purpose? I can also change the client first in another PR, and then refactor this PR on top of that.

Reply:
Making it in this PR works for me.